packages/server/lib/util/resolve.js

```diff
@@ -21,7 +20,7 @@ module.exports = {
 
       debug('resolving typescript with options %o', options)
```

Comment:
Should probably be updated to just note from where the resolve is run, the `options` object itself isn't used for anything else

Reply:
Updated